ssas/service/server.go

```diff
@@ -84,15 +81,15 @@ func ChooseSigningKey(signingKeyPath, signingKey string) (*rsa.PrivateKey, error
 
 // NewServer correctly initializes an instance of the Server type.
 func NewServer(name, port, version string, info interface{}, routes *chi.Mux, notSecure bool, useMTLS bool, signingKey *rsa.PrivateKey, ttl time.Duration, clientAssertAud string) *Server {
-
+	logger := log2.GetCtxLogger(context.Background())
```

Comment:
can we replace the [package](https://github.com/CMSgov/bcda-ssas-app/blob/95aad33ee6b1eb550bad7dc5a77c94939af2f78d/ssas/servi) `log` altogether in this file and just use `github.com/CMSgov/bcda-ssas-app/log` instead?

That way, we can rename `log2` and we are having an additional logger from the standard library also creating log events.